src/cookbooks/looker/performance_caching.md

```diff
@@ -0,0 +1,75 @@
+# Optimizing Looker Performance - Caching
+
+This how-to guide is about improving performance of Looker dashboards and explores using caching. This is particularly
+useful if you have many users accessing a dashboard or explore, especially across timezones.
+
+This guide has two alternatives: Applying Generated Datagroups and Custom Datagroups
+
+> ⚠️ Some steps in this guide require knowledge of LookML and a developer license - ask in #data-help for assistance if
+> needed.
+
+The default setting in Looker is to store (cache) the results of any query for one hour. The majority of our data
+is updated at least daily. This means that while users will get the same result for a query, they might have to wait
```

Comment:
Probably more specific/accurate?

`This means that users who run the same query through a Dashboard or Explore, still have to wait`